README.md

```diff
@@ -145,23 +147,20 @@ Feature flags rely on .NET Core dependency injection. We can register the featur
 
 ``` C#
 using Microsoft.FeatureManagement;
-using Microsoft.FeatureManagement.FeatureFilters;
 
 public class Startup
 {
   public void ConfigureServices(IServiceCollection services)
   {
       services.AddFeatureManagement()
-              .AddFeatureFilter<PercentageFilter>()
-              .AddFeatureFilter<TimeWindowFilter>();
+              .AddFeatureFilter<MyCustomizedFilter>();
   }
 }
 ```
 
-This tells the feature manager to use the "FeatureManagement" section from the configuration for feature flag settings. It also registers two built-in feature filters named `PercentageFilter` and `TimeWindowFilter`. When filters are referenced in feature flag settings (appsettings.json) the _Filter_ part of the type name can be omitted.
-
+This tells the feature manager to use the "FeatureManagement" section from the configuration for feature flag settings. It also registers a feature filter named `MyCustomizedFilter`. When filters are referenced in feature flag settings (appsettings.json) the _Filter_ part of the type name can be omitted.

```

Comment:
I'm thinking, perhaps we should omit the AddFeatureFilter part and leave that to the custom filters section.